core/src/main/java/org/jboss/pnc/build/finder/koji/KojiJSONUtils.java

```diff
@@ -26,6 +26,9 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 public final class KojiJSONUtils {
+    // Shared instance of BuildFinderObjectMapper.
+    public static final BuildFinderObjectMapper BUILD_FINDER_OBJECT_MAPPER = new BuildFinderObjectMapper();
```

Comment:
OK, I guess I should search for other instances now and make them use this one.

Reply:
:+1: